tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -4949,15 +5139,9 @@ public function theDownloadedPreviewContentShouldMatchWithFixturesPreviewContent
 	 * @return void
 	 */
 	public function userDownloadsThePreviewOfWithWidthAndHeight(string $user, string $path, string $width, string $height):void {
-		$response = $this->downloadPreviews(
-			$user,
-			$path,
-			null,
-			$width,
-			$height
-		);
-		$this->theHTTPStatusCodeShouldBe(200, "", $response);
-		$this->imageDimensionsShouldBe($width, $height, $response);
+		$response = $this->downloadPreviewOfFiles($user, $path, $width, $height);
+		$this->theHTTPStatusCodeShouldBe(200, '', $response);
+		$this->imageDimensionsShouldBe($width, $height);
```

Comment:
Existing one is correct I guess
```suggestion
```

Reply:
This could be the reason for different response status code.